src/annotator/integrations/html.js

```diff
@@ -13,6 +13,11 @@ import { scrollElementIntoView } from '../util/scroll';
  * @typedef {import('../../types/annotator').SidebarLayout} SidebarLayout
  */
 
+// When activating side-by-side mode, make sure there is at least this amount
+// of space (in pixels) left for the document's content. Any narrower and the
+// content line lengths and scale are too short to be readable.
+const MIN_HTML_WIDTH = 576;
```

Comment:
Perhaps we could compromise in the middle. I'm hesitant to conflate the user context on a narrow-viewport handheld device with a wider viewport at arm's or more distance based on (CSS/virtual) pixels alone—I think we're wading into apples-and-oranges territory. Because we are not giving the user an option here, I'd also argue for conservatism.

If I put myself in the shoes of a student user trying to annotate content within the confines of a Canvas iframe, I don't really want to be trying to read content at the widths of any of these screen shots.

Perhaps we could compromise for now at 480px?

Reply:
> Because we are not giving the user an option here, I'd also argue for conservatism.

The user can close the sidebar, as they already have to do today, since they can't read the content anyway if it is substantially under the sidebar. In my mind the main reason for still having _some_ threshold was because at some point the page layout will completely break, and so the value of being able to see the content alongside the sidebar drops below a threshold of being useful.

> Perhaps we could compromise for now at 480px?

OK. Let's go with this. We can easily revise it later.